policyengine_us/variables/gov/states/mt/tax/income/mt_taxable_income_joint.py

```diff
@@ -0,0 +1,29 @@
+from policyengine_us.model_api import *
+
+
+class mt_taxable_income_joint(Variable):
+    value_type = float
+    entity = Person
+    label = "Montana taxable income when married couples file jointly"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.MT
+    reference = (
+        "https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/05/Montana-Idividiual-Income-Tax-Return-Form-2-2022v6.2.pdf#page=1",
+        "https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2022/12/Form-2-2022-Instructions.pdf#page=16",
+    )
+
+    def formula(person, period, parameters):
+        is_head = person("is_tax_unit_head", period)
+        agi = person("mt_agi", period)
+        head_agi = is_head * person.tax_unit.sum(agi)
```

Comment:
just checking this shouldn't be this? ie should this be zero for spouses? (please add a comment and test for this if you don't have one)
```suggestion
        head_agi = person.tax_unit.sum(is_head * agi)
```

Reply:
This is representing total AGI which is allocated to the head rather than head AGI 
```suggestion
        total_agi = is_head * person.tax_unit.sum(agi)
```
This seems to be the appropriate fix to avoid confusion